tools/evmc/main.cpp

```diff
@@ -6,9 +6,24 @@
 #include <CLI/CLI.hpp>
 #include <evmc/hex.hpp>
 #include <evmc/loader.h>
+#include <fstream>
 
 namespace
 {
+/// Returns the input str if already valid hex string. Otherwise, interprets the str as a file
+/// name and loads the hex string from this file.
+/// @todo The file content is not validated.
```

Comment:
So does the file needs to be a hex string, or it is a binary file?

Reply:
hex string

Reply:
Can it be validated? Even if just validating it here with `abort`, I know it is not nice, but reduces confusion should someone misuse it.

Reply:
It still throws "invalid hex digit" exception or similar. This is caught and reported although it is not clear where the error comes from. I added test for this case.

Reply:
Isn't it coming from somewhere inside `cmd::run`?

Reply:
It is. But I mean user will see the error about invalid hex without information about the source. He has to guess it is code and/or input what is invalid.

Reply:
I think it comes from there as it uses `load_hex` which throws the error and `main` catches that. Can you leave this as a comment here (that it is caught later in `evmc::run`), for now?